aea/cli/run.py

```diff
@@ -195,5 +197,14 @@ def _build_aea(
         )
         aea = builder.build(connection_ids=connection_ids, password=password)
         return aea
+    except AEAWalletNoAddressException:
+        error_msg = (
+            "You haven't specified any private key for the AEA project.\n"
+            "Please add one by using the following commands:\n"
+            "\n"
+            f"    aea generate-key {DEFAULT_LEDGER}\n"
+            f"    aea add-key fetchai {PRIVATE_KEY_PATH_SCHEMA.format(DEFAULT_LEDGER)}\n"
```

Comment:
Hm, not sure it makes sense to tie this to default. Why not just say:
`Please add one by using the command `aea genera-key` and `aea add-key` for the ledger of your choice.`?

Reply:
ok